packages/frontend/app/routes/tenants.$tenantId.tsx

```diff
@@ -0,0 +1,333 @@
+                  <PasswordInput
+                    name='apiSecret'
+                    label='API Secret'
+                    defaultValue={tenant.apiSecret ?? undefined}
+                    required
+                  />
+                </div>
+                <div className='flex justify-end p-4'>
+                  <Button
+                    aria-label='save sensitive information'
+                    type='submit'
+                    name='intent'
+                    value='sensitive'
+                  >
+                    {isSubmitting ? 'Saving ...' : 'Save'}
+                  </Button>
+                </div>
+              </fieldset>
+            </Form>
+          </div>
+        </div>
+        {/* Sensitive - END */}
+        {/* Identity Provider Information */}
+        <div className='grid grid-cols-1 py-3 gap-6 md:grid-cols-3 border-b border-pearl'>
+          <div className='col-span-1 pt-3'>
+            <h3 className='text-lg font-medium'>
+              Identity Provider Information
+            </h3>
+            <ErrorPanel errors={response?.errors.general.message} />
+          </div>
+          <div className='md:col-span-2 bg-white rounded-md shadow-md'>
+            <Form method='post' replace preventScrollReset>
+              <fieldset disabled={isSubmitting}>
+                <div className='w-full p-4 space-y-3'>
+                  <Input type='hidden' name='id' value={tenant.id} />
+                  <Input
+                    name='idpConsentUrl'
+                    label='Consent URL'
+                    defaultValue={tenant.idpConsentUrl ?? undefined}
+                  />
+                  <PasswordInput
+                    name='idpSecret'
+                    label='Secret'
+                    defaultValue={tenant.idpSecret ?? undefined}
+                  />
+                </div>
+                <div className='flex justify-end p-4'>
+                  <Button
+                    aria-label='save ip information'
+                    type='submit'
+                    name='intent'
+                    value='ip'
+                  >
+                    {isSubmitting ? 'Saving ...' : 'Save'}
+                  </Button>
+                </div>
+              </fieldset>
+            </Form>
+          </div>
+        </div>
+        {/* Identity Provider Information - END */}
+
+        {/* DELETE TENANT - Danger zone */}
+        <DangerZone title='Delete Tenant'>
+          <Form method='post' onSubmit={submitHandler}>
+            <Input type='hidden' name='id' value={tenant.id} />
+            <Input type='hidden' name='intent' value='delete' />
+            <Button
+              disabled={!isOperator}
+              type='submit'
+              intent='danger'
+              aria-label='delete tenant'
+            >
+              Delete tenant
+            </Button>
+          </Form>
+        </DangerZone>
```

Comment:
I think we should hide this completely for non operators

Reply:
Fixed.